### PR TITLE
se/error_logs

### PR DESCRIPTION
**Pull request by svenefftinge**
Fix various error logs

 - https://console.cloud.google.com/errors/CNm39pOJiIjWXg?time=P7D&project=gitpod-191109
 - https://console.cloud.google.com/errors/CNHRm-emwP24twE?time=P7D&project=gitpod-191109

```release-notes
NONE
```

### CODE REVIEW

**Comment by roboquat** [comment]
@svenefftinge: Adding the "do-not-merge/release-note-label-needed" label because no release-note block was detected, please follow our [release note process](https://git.k8s.io/community/contributors/guide/release-notes.md) to remove it.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: components/server/ee/src/prebuilds/github-app.ts [hunk 1]
```diff
@@ -359,14 +359,12 @@ export class GithubApp {
         //
         const installation = await this.appInstallationDB.findInstallation("github", String(installationId));
         if (!installation) {
-            log.error("Prebuilt requested from unknown GitHub app installation");
```
Comment (geropl) [inline components/server/ee/src/prebuilds/github-app.ts:362]: Why not turn this into debug/trace? IMO log volume is no argument here (called very seldomly).

File: components/gitpod-db/src/typeorm/accounting-db-impl.ts [hunk 2]
```diff
@@ -123,9 +123,10 @@ export class TypeORMAccountingDBImpl implements AccountingDB {
         return await (await this.getSubscriptionRepo()).delete(subscription as DBSubscription);
     }
 
-    async findActiveSubscriptionByPlanID(planID: string): Promise<Subscription[]> {
+    async findActiveSubscriptionByPlanID(planID: string, date: string): Promise<Subscription[]> {
         return (await this.getSubscriptionRepo()).createQueryBuilder('subscription')
             .where('subscription.planID = :planID', { planID })
+            .andWhere('subscription.startDate <= :date AND (subscription.endDate = "" OR subscription.endDate > :date)', { date: date })
```
Comment (geropl) [inline components/gitpod-db/src/typeorm/accounting-db-impl.ts:129]: :+1:

File: components/server/ee/src/prebuilds/prebuilt-status-maintainer.ts [hunk 3]
```diff
@@ -100,6 +100,8 @@ export class PrebuildStatusMaintainer implements Disposable {
     protected getConclusionFromPrebuildState(pws: PrebuiltWorkspace): "error" | "failure" | "pending" | "success" {
         if (pws.state === "aborted") {
             return "error";
+        } else if (pws.state === "queued") {
```
Comment (geropl) [inline components/server/ee/src/prebuilds/prebuilt-status-maintainer.ts:103]: TIL this is used for reporting status back to GitHub only, and has effectively no impact (besides being easier on logging), so: :+1:

**Review by geropl — APPROVED** [review]
LGTM

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: 5001be4c7d9d8a489215e6cb10e915060b2bfa21</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/5575#pullrequestreview-748814197" title="Approved">geropl</a>*, *<a href="https://github.com/gitpod-io/gitpod/pull/5575#issuecomment-914342468" title="Approved">svenefftinge</a>*

Associated issue requirement bypassed by: *<a href="https://github.com/gitpod-io/gitpod/pull/5575#issuecomment-914342468" title="Approved">svenefftinge</a>*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[components/ee/payment-endpoint/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ee/payment-endpoint/OWNERS)~~ [geropl,svenefftinge]
- ~~[components/gitpod-db/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/gitpod-db/OWNERS)~~ [geropl,svenefftinge]
- ~~[components/server/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/server/OWNERS)~~ [geropl,svenefftinge]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**